spyder/widgets/collectionseditor.py

```diff
@@ -1132,57 +1133,41 @@ def view_item(self):
         index = index.child(index.row(), 3)
         self.delegate.createEditor(self, None, index, object_explorer=True)
 
-    def __prepare_plot(self):
-        try:
-            import guiqwt.pyplot   #analysis:ignore
-            return True
-        except:
-            try:
-                if 'matplotlib' not in sys.modules:
-                    import matplotlib  # noqa
-                return True
-            except Exception:
-                QMessageBox.warning(self, _("Import error"),
-                                    _("Please install <b>matplotlib</b>"
-                                      " or <b>guiqwt</b>."))
-
     def plot_item(self, funcname):
         """Plot item"""
         index = self.currentIndex()
-        if self.__prepare_plot():
-            if self.proxy_model:
-                key = self.source_model.get_key(
-                    self.proxy_model.mapToSource(index))
-            else:
-                key = self.source_model.get_key(index)
-            try:
-                self.plot(key, funcname)
-            except (ValueError, TypeError) as error:
-                QMessageBox.critical(self, _( "Plot"),
-                                     _("<b>Unable to plot data.</b>"
-                                       "<br><br>Error message:<br>%s"
-                                       ) % str(error))
+        if self.proxy_model:
+            key = self.source_model.get_key(
+                self.proxy_model.mapToSource(index))
+        else:
+            key = self.source_model.get_key(index)
+        try:
+            self.plot(key, funcname)
+        except (ValueError, TypeError) as error:
+            QMessageBox.critical(self, _( "Plot"),
+                                    _("<b>Unable to plot data.</b>"
+                                    "<br><br>Error message:<br>%s"
+                                    ) % str(error))
```

Comment:
```suggestion
            QMessageBox.critical(
                self,
                _( "Plot"),
                _("<b>Unable to plot data.</b><br><br>"
                  "The error message was:<br>%s") % str(error)
             )
```

Improve formatting and message a bit.

Reply:
Regarding code formatting, you really should consider using [Black ](https://black.readthedocs.io/en/stable/) and force everyone contributing to do so. At work, all my teams are using [Black](https://black.readthedocs.io/en/stable/) together with [isort](https://pycqa.github.io/isort/) with proper configuration (I gradually imposed this on all projects since 2018). [CodraFT](https://codra-ingenierie-informatique.github.io/CodraFT/) is an example of project following the standards in terms of code quality.

Using (and forcing all contributors to use) [Black ](https://black.readthedocs.io/en/stable/) saves time, increases code quality and helps focusing on real code changes (instead of being annoyed with code appearance).

Reply:
Should I push this further? (by integrating [Black](https://black.readthedocs.io/en/stable/) into Spyder for example?)